apps/condo/domains/ticket/schema/TicketShareService.js

```diff
@@ -0,0 +1,55 @@
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const { COUNTRIES, RUSSIA_COUNTRY } = require('@condo/domains/common/constants/countries')
+const { SHARE_TICKET_MESSAGE_TYPE } = require('@condo/domains/notification/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { Ticket } = require('@condo/domains/ticket/utils/serverSchema')
+
+const TicketShareService = new GQLCustomSchema('TicketShareService', {
+    types: [
+        {
+            access: true,
+            type: 'input TicketShareInput { sender: JSON!,  users: [ID!]!, ticketId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type TicketShareOutput { status: String! }',
+        },
+    ],
+    mutations: [
+        {
+            access: true,
+            schema: 'ticketShare(data: TicketShareInput!): TicketShareOutput',
```

Comment:
No tests for service

Reply:
we need at least permissions tests
1. Guest is not allowed to share ticket
2. The User from this organization can share ticket
3. The User from another organization can not share a ticket

Reply:
The name of mutations should be a verb